DO NOT MERGE - refactor(curriculum): refactoring RPG project to prepare for the addition of the intro JS project (#54321)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-dom-manipulation-by-building-a-role-playing-game/62a3b49686792938718b90d3.md

---
id: 62a3b49686792938718b90d3
title: Step 8
challengeType: 0
dashedName: step-8
---

# --description--

Your player's inventory will hold multiple weapons. For now, you want the player to start with just a `"stick"`. 

> Assign an array to the `inventory` variable that has the string `"stick"` as its only value.

Reviewer: ```suggestion
Assign an array to the `inventory` variable with the string `"stick"` as its only value.
```
